### PR TITLE
Clean docstring of CountourSet

### PR DESCRIPTION
**Pull request by dstansby**

### CODE REVIEW

File: lib/matplotlib/contour.py [hunk 1]
```diff
@@ -757,44 +792,6 @@ class ContourSet(cm.ScalarMappable, ContourLabeler):
     """
 
     def __init__(self, ax, *args, **kwargs):
-        """
```
Comment (dstansby) [inline lib/matplotlib/contour.py]: numpydoc says to put the parameters section in the class docstring (not the ``__init__`` docstring, before the attributes doc, which I think makes sense, and makes everything look a bit more sensible.

**Comment by dstansby** [comment]
Before: https://matplotlib.org/devdocs/api/contour_api.html?highlight=contourset#matplotlib.contour.ContourSet

After: https://782-74355158-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/api/contour_api.html?highlight=contourset#matplotlib.contour.ContourSet

**Comment by jklymak** [comment]
@meeseeksdev backport to v2.2-doc

**Comment by jklymak** [comment]
@meeseeksdev backport to v2.2.2-doc

**Pull request merged**